src/components/UnovaSVG.html

```diff
@@ -0,0 +1,705 @@
+<g data-bind='if: player.region == GameConstants.Region.unova'>
+    <image width="100%" xlink:href="assets/images/unova.png"></image>
+
+    <!-- ------ -->
+    <!-- ROUTES -->
+    <!-- ------ -->
+
+    <!--Route 1-->
+    <%- include('templates/mapRoute',
+        { route: 1, region: 4
+        , width: 3, height: 3
+        , x: 87, y: 51
+        , rotate: true
+        })
+    %>
+
+    <!--Route 2-->
+    <%- include('templates/mapRoute',
+        { route: 2, region: 4
+        , width: 4, height: 3
+        , x: 87, y: 43
+        , rotate: true
+        })
+    %>
+
+    <!--Route 3-->
+    <%- include('templates/mapRoute',
+        { route: 3, region: 4
+        , width: 6, height: 3
+        , x: 80, y: 40
+        })
+    %>
+
+    <!--extra Route 3 1-->
+    <%- include('templates/mapRoute',
+        { route: 3, region: 4
+        , width: 9, height: 3
+        , x: 64, y: 40
+        , rotate: true
+        , noText: true
+        })
+    %>
+
+    <!--extra Route 3 2-->
+    <%- include('templates/mapRoute',
+        { route: 3, region: 4
+        , width: 5, height: 3
+        , x: 67, y: 40
+        , noText: true
+        })
+    %>
+
+    <!--Route 4-->
+    <%- include('templates/mapRoute',
+        { route: 4, region: 4
+        , width: 12.5, height: 3
+        , x: 51, y: 35
+        , rotate: true
+        })
+    %>
+    <!--extra Route 4-->
+    <%- include('templates/mapRoute',
+        { route: 4, region: 4
+        , width: 10, height: 3
+        , x: 57, y: 49
+        , noText: true
+        })
+    %>
+
+    <!--Desert Resort-->
+    <%- include('templates/mapRoute',
+        { route: Desert Resort, region: 4
```

Comment:
The map crashes right now because this is an invalid object
```suggestion
        { route: "Desert Resort", region: 4
```